cmd/connectivity_pod_to_pod.go

```diff
@@ -5,17 +5,16 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/openservicemesh/osm-health/pkg/connectivity"
-	"github.com/openservicemesh/osm-health/pkg/kuberneteshelper"
+	"github.com/openservicemesh/osm-health/pkg/kubernetes/pod"
 )
 
 const connectivityPodToPodDesc = `
-Checks connectivity between Kubernetes pods
-	(add more descriptive description)
+Checks connectivity between two Kubernetes pods
 `
 
 const connectivityPodToPodExample = `
 Example:
-	(add example)
+	$ osm-health connectivity pod-to-pod namespace-a/pod-x namespace-b/pod-b
```

Comment:
```suggestion
	$ osm-health connectivity pod-to-pod namespace-a/pod-x namespace-b/pod-y
```

Reply:
Thanks for the hint!  Changed this to namespace-a/pod-a  and namespace-b/pod-b for clarity